web/viewer.js

```diff
@@ -171,8 +171,10 @@ function getViewerConfiguration() {
 function webViewerLoad() {
   var config = getViewerConfiguration();
   if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) {
-    require.config({paths: {'pdfjs': '../src', 'pdfjs-web': '.'}});
-    require(['pdfjs-web/app', 'pdfjs-web/pdf_print_service'], function (web) {
+    Promise.all([SystemJS.import('pdfjs-web/app'),
+                 SystemJS.import('pdfjs-web/pdf_print_service')])
+           .then(function (modules) {
+      var web = modules[0];
```

Comment:
Can we call this variable `app` instead? It's a bit confusing because `web` for me refers to the entire `web` folder, whereas `app` is clearly only `app.js`.